fjord/feedback/views.py

```diff
@@ -135,10 +142,10 @@ def _handle_feedback_post(request, locale=None, product=None,
     data = form.cleaned_data
 
     description = data.get('description', u'').strip()
+
     if not description:
-        # If there's no description, then there's nothing to do here,
-        # so thank the user and move on.
-        return HttpResponseRedirect(reverse('thanks'))
+        # If there's no description populate a generic one
+        description = u'Sentiment: %s' % (('Sad', 'Happy')[data['happy']])
```

Comment:
I don't love this approach, but it did seem like the least disruptive. The screenshots hopefully explain why we need it: When Fennec users say "It made me happy," they don't get an input form. A couple thoughts on this:
1) I don't think we would want that input form to be required, even if we did include it. It's simply not as likely to attract description content due to thumbtyping. 
2) I don't think we should include it. Our hope is to send happy mobile users to the application store where they can give us 5 stars and create a growth loop. 

Considering those, a couple ideas. Which do you like better?

a) Drop "happy" mobile feedback on the floor. Use the form only as a way route mobile users to the app store.
b) Help me dig into the bigger changes that would allow us to capture happy mobile feedback without requiring description content.

I think the answer probably depends on Input's overarching reason to exist. Is it to capture all product feedback, or just qualitative feedback?